### ISSUE TITLE
Consider switching continuous builds to 12.2-RELEASE

### ISSUE THREAD

**Issue by probonopd** [furybsd/furybsd-livecd#241]
__tl;dr: Xorg is broken in 12.2-RELEASE, unless we get the 12.2 kernel related packages built on 12.2.__

I am interested in 12.2 because it contains an improved Linuxulator, which I need for [debugging Linux AppImages using Linux `strace`](https://github.com/AppImage/AppImageKit/issues/98#issuecomment-716037952). Hence I built a variant based on 12.2-RC3, but the Intel GPU driver is not working, resulting in broken Xorg. Looks like it is a known (to @vermaden, anyway) issue:

https://twitter.com/vermaden/status/1321375859429773312

> Dear #FreeBSD users ...
>
> Please be advised that kernel related packages like:
> - drm-fbsd12.0-kmod
> - drm-legacy-kmod
> - drm-kmod
> 
> ... are BROKEN on 12.2-RELEASE (until 12.1-RELEASE is supported) and need to be (re)built from #FreeBSD Ports.

First I thought there must have been a typo in the version numbers, until I saw

https://twitter.com/vermaden/status/1321455159088734208

> That is one of strange pitfalls of #FreeBSD releases ...
>
>I can not understand why FreeBSD developers insist on building 12.2 kernel related packages on 12.1 so they will not work on 12.2 systems ...
>
> Its just literally ASKING for problems and discouraging people from FreeBSD.

Here are my questions:

* How come that 12.1 kernel related packages were working when 12.1 was the latest in the 12.x line? Were they built on 12.0 or 12.1? If they were built on 12.0, why did they work on 12.1 whereas those built on 12.1 refuse to work on 12.2?
* Does this mean we have to wait until FreeBSD 12.3 is out in order to get working GPUs on 12.2? That makes the latest dot release pretty much broken for desktop users (unless one is willing to compile those drivers from ports)
* Wouldn't it be awesome if the FreeBSD base system would contain the drivers needed for GPUs? It's not the 80s anymore, after all. Who (besides embedded system developers) is really using FreeBSD on a non-graphical console these days?
* If that is absolutely not possible, why not build the 12.2 kernel related packages on 12.2?

In the meantime:

https://twitter.com/nomadlogicLA/status/1321468753578635265

> One thing that’s helpful is doing a “make package” rather than “make install” - then you can install the resulting pkg file (and lock it from upgrades) using your normal pkg workflow. Heck you can even host it on a web server for multiple systems to install

* Does not sound impossible to do, not even too difficult. Am I missing something?
* Could someone compile all 12.2 kernel related packages on 12.2 and provide them in a package repository?
* If "someone" can do this, why can the FreeBSD project not do it?

This whole situation somehow feels like the FreeBSD project couldn't care less about desktop users. And it certainly violates POLA...
